### PR TITLE
Fix fatal error on stage.

### PR DESCRIPTION
**Pull request by angaither**
Refs #5910

### CODE REVIEW

File: lib/modules/dosomething/dosomething_campaign/dosomething_campaign.helpers.inc [hunk 1]
```diff
@@ -624,8 +624,8 @@ function dosomething_campaign_get_scholarship($nid) {
  * @return string
  *  Formatted time (e.g. "4 days left")
  */
-function dosomething_campaign_get_hot_time_left(&$vars) {
-  $today = $vars['hm_today'];
+function dosomething_campaign_get_hot_time_left($node, &$vars = NULL) {
```
Comment (sbsmith86) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.helpers.inc:627]: Do you end up using `$node` anywhere?

Comment (angaither) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.helpers.inc:627]: `$node` is used [here](https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_campaign/dosomething_campaign_hot_shares/dosomething_campaign_hot_shares.cron.inc#L30)

**Comment by sbsmith86** [comment]
Looks good! Except, I still don't see the `$node` parameter being used anywhere in the `dosomething_campaign_get_hot_time_left()` function. Does the cron get called somewhere in this functions?

**Comment by angaither** [comment]
@sbsmith86 yeah it's passed in in the cron, we might need to update that later today

**Comment by angaither** [comment]
yeah cron errors, I'll make an issue for that :wink:

File: lib/modules/dosomething/dosomething_campaign/dosomething_campaign.helpers.inc (hunk 1 above)
Comment (sbsmith86) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.helpers.inc:627]: @angaither right, but the actual function doesn't ever use it. It uses `$vars`.

**Pull request merged**